### PR TITLE
Update Kinto version / Edit a form / uiSchema download

### PR DESCRIPTION
**Pull request by adamjhamer**
These were some basic changes we made which made a great deal of difference to us.

### CODE REVIEW

File: formbuilder/containers/builder/FormEdit.js [hunk 1]
```diff
@@ -0,0 +1,29 @@
+import React, { Component } from "react";
+import FormContainer from "../../containers/builder/FormContainer";
+import {getFormID} from "../../utils";
+
+export default class FormEdit extends Component {
+
+  componentDidMount() {
+    // If the schema is empty, then try to load the schema from the
```
Comment (almet) [inline formbuilder/containers/builder/FormEdit.js]: This comment is not finished :-)

File: README.md [hunk 2]
```diff
@@ -24,8 +24,12 @@ $ npm run start
 ```
 
 You also need to have a [Kinto](https://kinto.readthedocs.io) server **greater
-than 4.3.1** in order to store your data. If you don't already have one, follow [the
-installation instructions](http://kinto.readthedocs.io/en/stable/tutorials/install.html)!
+than 4.3.1** in order to store your data and **less than 11.0.0** for an OOTB
```
Comment (almet) [inline README.md]: nit: avoid abbreviations

**Review by almet — APPROVED** [review]
Thanks! Looks good to me. To merge once some stuff is fixed :-)

File: formbuilder/containers/builder/FormEdit.js (hunk 1 above)
Comment (adamjhamer) [reply formbuilder/containers/builder/FormEdit.js]: Looks like I did a copy/paste from UserForm.js!

**Pull request merged**